lib/tasks/manage.rake

```diff
@@ -95,6 +95,26 @@ namespace :manage do
     end
   end
 
+  def update_business_finder_subscriptions
+    business_finder_facet_group_content_id = "52435175-82ed-4a04-adef-74c0199d0f46".to_s.freeze
```

Comment:
`.to_s` is not necessary here, it's already a string.

Reply:
I had hard time with linter here, which wouldn't let me interpolate. 
When calling `find_by_links_value` the content_id has to be wrapped in a double quote so I want the string to be `"52435175-82ed-4a04-adef-74c0199d0f46"` not just `52435175-82ed-4a04-adef-74c0199d0f46`, if that makes sense. I could't think of a better way of doing it and keeping the linter happy. I will rename the variable to make it clearer.

Reply:
Oh ok, makes sense 👍 In that case I'd maybe add a comment to the code saying the string needs to have double-quotes on it.

Could that be done with something like this...?

```suggestion
    business_finder_facet_group_content_id = "\"52435175-82ed-4a04-adef-74c0199d0f46\"".freeze
```

Reply:
The other option would be to put the string here:
```
business_finder_facet_group_content_id = "52435175-82ed-4a04-adef-74c0199d0f46"
```

Then modify the value when invoking `find_by_links_value` (either with `to_s` or by escaping the quotes), e.g.
```
business_finder_subscriber_lists = SubscriberList.find_by_links_value(formatted_facet_group_content_id.to_s.freeze) - primary_business_finder_subscriber_list
```

It doesn't make much difference here since the value is only being used once, but this would be cleaner if the value for `business_finder_facet_group_content_id` were being reused elsewhere..